pkg/proxy/metrics/metrics.go

```diff
@@ -180,6 +208,10 @@ var registerMetricsOnce sync.Once
 func RegisterMetrics() {
 	registerMetricsOnce.Do(func() {
 		legacyregistry.MustRegister(SyncProxyRulesLatency)
+		if utilfeature.DefaultFeatureGate.Enabled(features.MinimizeIPTablesRestore) {
```

Comment:
Given that we're already in Beta as of now (and planning to go GA in 1.28), I suggest we just register this metric unconditionally (worst case if someone disables the feature they will have the metric not being updated).

@dgrisonnet - thoughts?

Reply:
yeah, I will not feature gate the metrics at all, nor here neither on pkg/proxy/iptables/proxier.go